system/zmodem/host/nuttx/config.h

```diff
@@ -34,8 +36,6 @@
 
 /* Environment stuff */
 
-#define OK 0
-#define ERROR -1
 #define FAR
 #define DEBUGASSERT assert
```

Comment:
Maybe we can get those definitions from compiler.h as well?

Reply:
no, since it is compiled in host environment.